index.js

```diff
@@ -33,17 +33,13 @@ class GrcSlack extends Base {
   }
 
   logError (reqChannel, err, ...extra) {
-    const parseError = e => {
```

Comment:
tbh I'm not sure why we're not using simply util.inspect to get full err with trace and params in string:
```
> const util = require('util')
undefined
> const err = new Error('FOO')
undefined
> err.code =505
505
> err
Error: FOO
    at REPL2:1:13
    at Script.runInThisContext (node:vm:123:12)
    at REPLServer.defaultEval (node:repl:569:29)
    at bound (node:domain:433:15)
    at REPLServer.runBound [as eval] (node:domain:444:12)
    at REPLServer.onLine (node:repl:899:10)
    at REPLServer.emit (node:events:526:35)
    at REPLServer.emit (node:domain:489:12)
    at [_onLine] [as _onLine] (node:internal/readline/interface:422:12)
    at [_line] [as _line] (node:internal/readline/interface:893:18) {
  code: 505
}
> util.inspect(err)
'Error: FOO\n' +
  '    at REPL2:1:13\n' +
  '    at Script.runInThisContext (node:vm:123:12)\n' +
  '    at REPLServer.defaultEval (node:repl:569:29)\n' +
  '    at bound (node:domain:433:15)\n' +
  '    at REPLServer.runBound [as eval] (node:domain:444:12)\n' +
  '    at REPLServer.onLine (node:repl:899:10)\n' +
  '    at REPLServer.emit (node:events:526:35)\n' +
  '    at REPLServer.emit (node:domain:489:12)\n' +
  '    at [_onLine] [as _onLine] (node:internal/readline/interface:422:12)\n' +
  '    at [_line] [as _line] (node:internal/readline/interface:893:18) {\n' +
  '  code: 505\n' +
  '}'
> 
> 
> err.stack
'Error: FOO\n' +
  '    at REPL2:1:13\n' +
  '    at Script.runInThisContext (node:vm:123:12)\n' +
  '    at REPLServer.defaultEval (node:repl:569:29)\n' +
  '    at bound (node:domain:433:15)\n' +
  '    at REPLServer.runBound [as eval] (node:domain:444:12)\n' +
  '    at REPLServer.onLine (node:repl:899:10)\n' +
  '    at REPLServer.emit (node:events:526:35)\n' +
  '    at REPLServer.emit (node:domain:489:12)\n' +
  '    at [_onLine] [as _onLine] (node:internal/readline/interface:422:12)\n' +
  '    at [_line] [as _line] (node:internal/readline/interface:893:18)'
> 
> 
> 
> util.inspect('fooo')
"'fooo'"
```